docker/Dockerfile_py27

```diff
@@ -30,18 +30,15 @@ FROM nipype/nipype:latest
 MAINTAINER The nipype developers https://github.com/nipy/nipype
 
 COPY docker/files/run_* /usr/bin/
-RUN chmod +x /usr/bin/run_*
+RUN chmod +x /usr/bin/run_* && rm /usr/bin/fsl_imglob.py
```

Comment:
The softlink of `${FSLDIR}/bin/imglob` to `/usr/bin/fsl_imglob.py` exists at this point. Not sure if we want to move the imglob replacement to the base image.